src/client/themes/default/index.ts

```diff
@@ -74,7 +74,7 @@ const sharedTheme: SharedTheme = {
     desktopL: 2560,
   },
   globalFont: '"Roboto", sans-serif',
-  globalMaxWidth: '120rem',
+  globalMaxWidth: '90%',
```

Comment:
want this to still be a constant. Prevents weird stuff on massive screens

Reply:
done